fs-index/README.md

```diff
@@ -10,6 +10,7 @@ The most important struct in this crate is `ResourceIndex` which comes with:
 
 - **Reactive API**
   - `update_all`: Method to update the index by rescanning files and returning changes (additions/deletions).
+  - `update_one`: Method to update the index by rescanning a single file (addition/deletion/modification).
```

Comment:
Should be mentioned in other section. "Reactive" means that we have a stream of updates and we react on these updates, even if we manually poll the stream. But this "cherry-pick" is completely manual approach, not reactive.

Reply:
Good point. What about the term "Selective API"?